src/clm_ros_wrapper/CLMWrapper.cpp

```diff
@@ -425,6 +425,8 @@ void ClmWrapper::callback(const sensor_msgs::ImageConstPtr& msgIn)
 
             double confidence = 0.5 * (1 - clm_model.detection_certainty);
 
+            cout << "certainty";
+
```

Comment:
Printing out random numbers in the middle of the code without context is not the best practice.. please remove it, or if you really need it, please wrap it with some text and a ROS_DEBUG macro